src/test/java/com/project/parkinglot/controller/VehicleControllerTest.java

```diff
@@ -0,0 +1,200 @@
+package com.project.parkinglot.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.project.parkinglot.base.BaseControllerTest;
+import com.project.parkinglot.builder.VehicleRequestBuilder;
+import com.project.parkinglot.model.Vehicle;
+import com.project.parkinglot.model.dto.request.Vehicle.VehicleRequest;
+import com.project.parkinglot.model.dto.request.parking_area.ParkingAreaUpdateRequest;
+import com.project.parkinglot.model.mapper.vehicle.VehicleEntityToVehicleMapper;
+import com.project.parkinglot.model.mapper.vehicle.VehicleRequestToVehicleMapper;
+import com.project.parkinglot.model.mapper.vehicle.VehicleToVehicleEntityMapper;
+import com.project.parkinglot.service.vehicle.impl.VehicleServiceImpl;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.testcontainers.shaded.com.google.common.base.Verify;
+
+import java.util.UUID;
+
+ class VehicleControllerTest extends BaseControllerTest {
+
+     @MockBean
+     private VehicleServiceImpl vehicleService;
+
+     private final VehicleToVehicleEntityMapper vehicleToVehicleEntityMapper =
```

Comment:
`VehicleController` class'ın herhangi bir mapper olmadığı için buradaki mapperlar silelim.

Reply:
Bu kısmı düzeltelim. @artfulCoder98